src/conways.rs

```diff
@@ -0,0 +1,126 @@
+const COLUMNS: usize = 32;
+const ROWS: usize = 32;
+
+#[derive(Clone, Copy, PartialEq, Debug)]
+pub enum CellState {
+    Alive,
+    Dead,
+}
+
+#[derive(Clone, Copy)]
+pub struct Position(pub usize, pub usize);
+
+pub struct Conways {
+    pub grid: Vec<Vec<CellState>>,
+}
+
+impl Conways {
+
+    // Using this function in the test
+    #[allow(dead_code)]
+    pub fn default() -> Self {
+        Self {
+            grid: vec![vec![CellState::Dead; COLUMNS]; ROWS]
+        }
+    }
```

Comment:
If this function is only used on tests maybe change the test and send and empty `vec` to `Conways::from()`

Reply:
Yes, so I changed the function in the test. Now I'm using `Conways::from` with an empty vec.

Reply:
Adding `#[cfg(test)]` at the top of the function also fixes this problem if you want to keep the default function while running tests